website/src/react-native/showcase.js

```diff
@@ -5,1128 +5,264 @@
+    infoTitle: 'Flipkart Buys Fashion E-tailer Myntra To Fight Amazon',
   },
   {
     name: 'SoundCloud Pulse',
     icon: 'https://i1.sndcdn.com/artworks-000149203716-k5je96-original.jpg',
-    link: 'https://itunes.apple.com/us/app/soundcloud-pulse-for-creators/id1074278256?mt=8',
-    author: 'SoundCloud',
-  },
-  {
-    name: 'Spero for Cancer',
-    icon: 'https://s3-us-west-1.amazonaws.com/cancerspot/site_images/Spero1024.png',
-    link: 'https://geo.itunes.apple.com/us/app/spero-for-cancer/id1033923573?mt=8',
-    author: 'Spero.io',
-    videos: [
-      'https://www.youtube.com/watch?v=JImX3L6qnj8',
-    ],
-  },
-  {
-    name: 'Squad',
-    icon: 'http://a4.mzstatic.com/us/r30/Purple69/v4/e8/5b/3f/e85b3f52-72f3-f427-a32e-a73efe2e9682/icon175x175.jpeg',
-    linkAppStore: 'https://itunes.apple.com/us/app/squad-snaps-for-groups-friends/id1043626975?mt=8',
-    linkPlayStore: 'https://play.google.com/store/apps/details?id=com.tackk.squad',
-    author: 'Tackk Inc.',
-    blogs: [
-      'https://medium.com/@rpastorelle/building-3-react-native-apps-in-one-summer-dcd0c31454ff#.z0fh9dehn',
-    ],
+    linkAppStore: 'https://itunes.apple.com/us/app/soundcloud-pulse-for-creators/id1074278256?mt=8',
+    infoLink: 'https://blog.soundcloud.com/2016/02/23/soundcloud-pulse-now-on-iphone/',
+    infoTitle: 'SoundCloud Pulse: now on iPhone',
   },
   {
     name: 'Start - medication manager for depression',
     icon: 'http://a1.mzstatic.com/us/r30/Purple49/v4/de/9b/6f/de9b6fe8-84ea-7a12-ba2c-0a6d6c7b10b0/icon175x175.png',
-    link: 'https://itunes.apple.com/us/app/start-medication-manager-for/id1012099928?mt=8',
-    author: 'Iodine Inc.',
+    linkAppStore: 'https://itunes.apple.com/us/app/start-medication-manager-for/id1012099928?mt=8',
+    infoLink: 'http://www.nytimes.com/2014/09/24/technology/to-gather-drug-information-a-health-start-up-turns-to-consumers.html?_r=0',
+    infoTitle: 'NYT: A Health Startup Turns to Consumers',
   },
   {
     name: 'Taxfyle - taxes filed on-demand via licensed CPA',
     icon: 'https://s3.amazonaws.com/taxfyle-public/images/taxfyle-icon-1024px.png',
-    link: 'https://itunes.apple.com/us/app/taxfyle/id1058033104?mt=8',
-    author: 'Taxfyle',
+    linkAppStore: 'https://itunes.apple.com/us/app/taxfyle/id1058033104?mt=8',
+    infoLink: 'http://www.techinsider.io/taxfyle-wants-to-be-the-uber-for-taxes-2016-4',
+    infoTitle: 'Taxfyle: the Uber for filing taxes',
   },
   {
     name: 'This AM',
     icon: 'http://s3.r29static.com//bin/public/efe/x/1542038/image.png',
-    link: 'https://itunes.apple.com/us/app/refinery29-this-am-top-breaking/id988472315?mt=8',
-    author: 'Refinery29',
-  },
-  {
-    name: 'Townske',
-    icon: 'http://a3.mzstatic.com/us/r30/Purple69/v4/8b/42/20/8b4220af-5165-91fd-0f05-014332df73ef/icon175x175.png',
-    link: 'https://itunes.apple.com/us/app/townske-stunning-city-guides/id1018136179?ls=1&mt=8',
-    author: 'Townske PTY LTD',
```

Comment:
Blogpost: https://hackernoon.com/townske-app-in-react-native-6ad557de7a7c#.2k579fync